libpromises/failsafe.cf

```diff
@@ -45,7 +45,7 @@ body agent control
       skipidentify => "true";
 
       # Bootstrapping can't continue without keys
-      abortclasses => { "no_ppkeys_ABORT" };
+      abortclasses => { "no_ppkeys_report_kept" };
```

Comment:
Why rename the abort class? It should be clearly visible that setting this will cause an ABORT! :-)